services/web-app/components/mdx/components.js

```diff
@@ -63,6 +63,9 @@ const components = {
   img: Image,
   table: PageTable,
   pre: Code,
+  code: ({ children }) => (
+    <mdxComponents.CodeSnippet type="inline">{children}</mdxComponents.CodeSnippet>
```

Comment:
Nit-picky, but should we pull this out into its own component?

Reply:
I definitely think we should! We could add a `Code` and `InlineCode` component to the mdx-components package. As we keep adding stuff over there, this file will get smaller to reflect that. Would that work? Or would you prefer to address it in this PR first and then move it to the mdx-components package as a second step?

Reply:
I'm fine with doing it when we move the Code component over to the mdx-components package.